src/clr-core/internal/utils/dom.ts

```diff
@@ -17,3 +17,25 @@ export function getElementWidthUnless(element: HTMLElement, unless: boolean) {
   }
   return '';
 }
+
+export type HTMLAttributeTuple = [string, string | boolean];
+
+export function setAttributes(element: HTMLElement, ...attributeTuples: HTMLAttributeTuple[]) {
```

Comment:
I like these functions. I'm going to modify my related functions to use `HTMLAttributeTuple` as well. 👍